src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/usecases/env/EnvVariableTest.kt

```diff
@@ -0,0 +1,80 @@
+package it.krzeminski.snakeyaml.engine.kmp.usecases.env
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.annotation.EnabledIf
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldEndWith
+import it.krzeminski.snakeyaml.engine.kmp.api.Load
+import it.krzeminski.snakeyaml.engine.kmp.api.LoadSettings
+import it.krzeminski.snakeyaml.engine.kmp.env.EnvConfig
+import it.krzeminski.snakeyaml.engine.kmp.exceptions.MissingEnvironmentVariableException
+import it.krzeminski.snakeyaml.engine.kmp.stringFromResources
+
+class EnvVariableTest : FunSpec({
+    // the variable is set by Gradle
+    val VALUE1 = "EnvironmentValue1"
+    val NULL_ENV_CONFIG = object : EnvConfig {
+        override fun getValueFor(name: String, separator: String?, value: String?, environment: String?): String? {
+            return null
+        }
+    }
+
+    test("Parse docker-compose.yaml example") {
```

Comment:
TODO: find a way to skip these for JS browser. Asked here: https://kotlinlang.slack.com/archives/CT0G9SD7Z/p1736115537677279

Reply:
Added an ugly try-catch.

Reply:
I think we can create a function like this:

Common:
```kotlin
expect fun isEnvUnsupported(): Boolean
```
js:
```kotlin
// returns 'true' is we are in the browser
actual fun isEnvUnsupported(): Boolean = js("return (typeof window !== 'undefined' && typeof document !== 'undefined')")
```

nonJs:
```kotlin
actual fun isEnvUnsupported(): Boolean = false
```

And this can be used to disable env tests in browser

Reply:
Good one, I was about to do something similar.
https://github.com/krzema12/snakeyaml-engine-kmp/pull/332/commits/569555d43743977b05ab1ee3b23421670576b031